src/components/calcite-input-time-picker/calcite-input-time-picker.tsx

```diff
@@ -81,6 +80,15 @@ export class CalciteInputTimePicker implements LabelableComponent {
   /** aria-label for the second up button */
   @Prop() intlSecondUp?: string;
 
+  /** BCP 47 language tag for desired language and country format */
+  @Prop({ attribute: "lang", mutable: true }) locale: string =
```

Comment:
I'm not sure that this is the right approach either because now the property and name won't match at all. Seems like this might just be a way to skirt around the warning/error. I think we should consult with Stencil on the best approach for listening to changes to native attributes on a component.

> `document.documentElement.lang || navigator.language || "en";`

I think it would be best if we don't set a default for this property. We can fallback to querying for the closest lang => navigator.language => "en".

Reply:
> I think we should consult with Stencil on the best approach for listening to changes to native attributes on a component.

I did just that.  Here's the [Stencil Slack thread](https://stencil-worldwide.slack.com/archives/C79EANFL7/p1636133062272300):

![image](https://user-images.githubusercontent.com/821864/140589825-c749ffd5-b7f9-45bd-9faf-b455c9e73743.png)

Reply:
Can we use `@internal` to avoid documenting the prop?